[Bug] Fix golangci-lint G107: Potential HTTP request made with variable url (gosec)

#### Current Behavior
<!-- A brief description of the issue. -->

```
internal/config/releases.go:102:14: G107: Potential HTTP request made with variable url (gosec)
	res, err := http.Get(shaURL)
```
#### Expected Behavior
<!-- A brief description of what you expected to happen. -->

`golangci-lint` should never fail. If a violation is acceptable, the code should be annotated so that the linter ignores it.

--- a/internal/config/releases.go
+++ b/internal/config/releases.go
@@ -99,7 +99,7 @@ func GetFileNames(url string, path string) ([]string, error) {
 
 func _getname(shaURL string, bpath []string) ([]string, error) {
 	var ans []string
-	res, err := http.Get(shaURL)
+	res, err := http.Get(shaURL) // #nosec
 	if err != nil {
 		return []string{}, err
 	}

